README.md

```diff
@@ -74,6 +74,7 @@ Table of Contents
   * https://www.blockspring.com/ - Share scripts with anyone on your team: cross language and with spreadsheet users. Free for 5 million runs a month.
   * https://helpmonks.com/ - Shared inbox for teams - Free for open source projects and non-profit organizations.
   * http://typetalk.in/ - Share and discuss ideas with your team through instant messaging on the web or on your mobile.
+  * https://www.wunderlist.com/ - Share your lists and work collaboratively on projects with your colleagues - Available for free on iPhone,iPad,Mac,Android,Windows and the Web.
```

Comment:
do you have a link that shows what the terms of the free trial is? stays free with limitations or time based? every link that I tried to get more details seem to insist on a signup